packages/hash/frontend/src/blocks/page/BlockView.tsx

```diff
@@ -216,49 +217,58 @@ export class BlockView implements NodeView<Schema> {
     const blockDraftId = this.getBlockDraftId();
 
     this.renderPortal(
-      <BlockViewContext.Provider value={this}>
-        <CollabPositionIndicators blockEntityId={blockEntityId} />
-        <BlockHighlight blockEntityId={blockEntityId} />
-        <BlockHandle
-          deleteBlock={this.deleteBlock}
-          entityStore={this.store}
-          draftId={blockDraftId}
-          ref={this.blockHandleRef}
-          onMouseDown={() => {
-            /**
-             * We only want to allow dragging from the drag handle
-             * so we set a flag which we can use to indicate
-             * whether a drag was initiated from the drag handle
-             *
-             * @todo we may not need this – we may be able to get
-             *       it from the event
-             */
-            this.allowDragging = true;
-
-            this.dragging = true;
-            this.dom.classList.add(styles["Block--dragging"]!);
-
-            const { tr } = this.editorView.state;
-
-            /**
-             * By triggering a selection of the node, we can ensure
-             * that the whole node is re-ordered when drag & drop
-             * starts
-             */
-            tr.setSelection(
-              NodeSelection.create<Schema>(
-                this.editorView.state.doc,
-                this.getPos(),
-              ),
-            );
-
-            this.editorView.dispatch(tr);
-
-            this.update(this.node);
-          }}
-          onClick={this.onDragEnd}
-        />
-      </BlockViewContext.Provider>,
+      <BlockContext.Consumer>
+        {(ctx) => {
+          return (
+            <BlockViewContext.Provider value={this}>
+              <BlockHighlight blockEntityId={blockEntityId} />
+              <CollabPositionIndicators blockEntityId={blockEntityId} />
```

Comment:
No, I think I resolved a conflict and this is how it ended up - restored the original order in https://github.com/hashintel/hash/pull/918/commits/ce795c2986dcec6762e4de383559df4651cc6137 so that no-one else wonders if it is significant